### ISSUE TITLE
Project progress using TDD (Suggest)

### ISSUE THREAD

**Issue by Junwon6** [Daniel-Liu-c0deb0t/Java-Machine-Learning#15]
I think there are many functions that we will implement.
However, it is difficult to identify what is broken each time it is updated.

I will introduce you to the TDD.
This is an implementation based on testing.
(link : https://en.wikipedia.org/wiki/Test-driven_development)

This method is used in many collaborative projects and I think it would be good if applied to this project.

If you want, I can help you create test cases for what you have already implemented.

**Comment by Daniel-Liu-c0deb0t** [comment]
Yes! Adding tests is very important, though I never got around to doing it. If you want to, you can add some.

Also, one day I'll figure out how to use Travis CI or something else to automatically compile and test on GitHub.
